pkg/compiler/test/policy_compiler_test.go

```diff
@@ -0,0 +1,140 @@
+package compiler_test
+
+import (
+	"errors"
+	"strings"
+	"testing"
+
+	"github.com/infobloxopen/seal/pkg/compiler"
+	_ "github.com/infobloxopen/seal/pkg/compiler/rego"
+)
+
+func checkError(got, expected error, t *testing.T) bool {
+	if (got != nil) != (expected != nil) {
+		t.Fatalf("expecting an error: '%+v', but got: '%+v'", expected, got)
+	}
+	if (got != nil) && (expected != nil) {
+		if got.Error() != expected.Error() {
+			t.Fatalf("\nexpected =\t'%+v'\ngot =\t'%+v'", expected, got)
+		} else {
+			return true
+		}
+	}
+	return false
+}
+func TestBackend(gt *testing.T) {
+	swaggerContent := strings.ReplaceAll(`openapi: "3.0.0"
+components:
+	schemas:
+		products.inventory:
+			type: object
+			x-seal-actions:
+			- allow
+			- deny
+			x-seal-verbs:
+			- inspect
+			- use
+			- manage
+			x-seal-default-action: deny
+		company.personnel:
+			type: object
+			x-seal-actions:
+			- allow
+			- deny
+			x-seal-verbs:
+			- inspect
+			- list
+			- manage
+			- operate
+			x-seal-default-action: deny
+`, "	", "  ")
+
+	tCases := map[string]struct {
+		compilerError  error
+		swaggerError   error
+		packageName    string
+		policyString   string
+		swaggerContent string
+		result         string
+	}{
+		"blank swagger": {
+			swaggerContent: " ",
+			swaggerError:   errors.New("swagger is required for inferring types"),
+		},
+		"no swagger actions": {
+			swaggerContent: "openapi: \"3.0.0\"\ncomponents:\n  schemas:",
+			swaggerError:   errors.New("Swagger error: no actions are defined"),
+		},
+		"policies errors": {
+			packageName: "products.errors",
+			policyString: `
+				allow subject group everyone to inspect fake;
+				`,
+			compilerError: errors.New("expected next token to be TYPE_PATTERN, got IDENT instead\nexpected next token to be user or group, got IDENT instead"),
```

Comment:
subsequent PR: this unit test is revealing an issue with the lexer/parser error - make error more user friendly:
```
evaluating `fake` - expected ...
```

Reply:
please don't fix this in this pr